### PR TITLE
Changed to version 4.1 and added an Only AMI count support

### PR DESCRIPTION
**Pull request by eliadeini**
### What does this PR do?

This PR changes titles of the 2 version 4.0  dashboards to version 4.1 and adds an Only-AMI count support.

### Motivation

What inspired you to submit this pull request?
Moving to a new CPM version + an Only AMI support bug

### Review checklist

- [ X ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ X ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by urseberry — COMMENTED** [review]
I don't have enough time to complete this review today, so it will go into the Docs team editorial review process. Someone will pick it up on Thursday afternoon. https://datadoghq.atlassian.net/browse/DOCS-3132

File: n2ws/README.md [hunk 1]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
 
 ### Installation
 
-1. Install the [Python integration][1].
+1.	##### Install the [Python Integration][1]
```
Comment (sarina-dd) [inline n2ws/README.md:15]:
```suggestion
1. Install the [Python Integration][1].
```

File: n2ws/README.md [hunk 2]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
...
 
-2. Enable support for Datadog on your N2WS instance:
+2.  ##### Enable support for Datadog on your N2WS instance:
```
Comment (sarina-dd) [inline n2ws/README.md:17]:
```suggestion
2.  Enable support for Datadog on your N2WS instance:
```

File: n2ws/README.md [hunk 3]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
...
     - Connect to your N2WS Backup and Recovery instance with SSH.
     - Add the lines below to `/cpmdata/conf/cpmserver.cfg`. You might need `sudo` privileges to perform this action.
         ```
         [external_monitoring]
         enabled=True
         ```
-    - Run `service apache2 restart`.
+    - Run ```service apache2 restart```.
+
+3.	##### Install the Datadog Agent on your N2WS Instance.
```
Comment (sarina-dd) [inline n2ws/README.md:26]:
```suggestion
3. Install the Datadog Agent on your N2WS Instance.
```

File: n2ws/README.md [hunk 4]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
...
+    Login to Datadog and go to Integrations -> Agent -> Ubuntu
+    Copy the agent ‘easy one-step install’ command 
+    Connect to your N2WS Backup and Recovery Instance with SSH. You may require `sudo` privileges to perform this action.
+
+4.	##### Setup Datadog Dashboard metrics
```
Comment (sarina-dd) [inline n2ws/README.md:31]:
```suggestion
4. Setup Datadog Dashboard metrics
```

File: n2ws/README.md [hunk 5]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
...
-    - **Over**: Select data from the list. All N2WS user data begins with the string `cpm:user:<USER_NAME>`. You can select a specific user or the entire N2WS instance.
+    **Over**: Select data from the list. All N2WS users data begin with the string ‘cpm:user:<user-name>’.
+              You can select either a specific user or the entire N2WS instance.
 
 
-5. Add N2WS dashboards to your Datadog account:
-    - Navigate to the [N2WS tile][3] and install the integration.
-    - Clicking the install button adds the dashboards: `N2WSBackup&Recovery-Graphicalversion`, `N2WSBackup&Recovery-Graphicalversion-areas`, and `N2WSBackup&Recovery-Squaresdashboard`.
-    - Alternatively, users can [import JSON templates from N2WS][4].
+5.	##### Get N2WS dashboards
```
Comment (sarina-dd) [inline n2ws/README.md:40]:
```suggestion
5. Get N2WS dashboards
```

File: n2ws/README.md [hunk 6]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
...
+    In [Datadog Integrations][3] , search for 'N2WS' tile and install it.
```
Comment (sarina-dd) [inline n2ws/README.md:41]:
```suggestion
    In [Datadog Integrations][3], search for 'N2WS' tile and install it. 
```

File: n2ws/README.md [hunk 7]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
...
+              You can select either a specific user or the entire N2WS instance.
 
 
-5. Add N2WS dashboards to your Datadog account:
-    - Navigate to the [N2WS tile][3] and install the integration.
-    - Clicking the install button adds the dashboards: `N2WSBackup&Recovery-Graphicalversion`, `N2WSBackup&Recovery-Graphicalversion-areas`, and `N2WSBackup&Recovery-Squaresdashboard`.
-    - Alternatively, users can [import JSON templates from N2WS][4].
+5.	##### Get N2WS dashboards
+    In [Datadog Integrations][3] , search for 'N2WS' tile and install it. 
+    You will get 5 types of dashboards to your account, 3 for N2WS Backup & Recovery v3.2.1 (only AWS supported) and 2 for for N2WS Backup & Recovery v4.1 (includes Azure):
```
Comment (sarina-dd) [inline n2ws/README.md:42]:
```suggestion
    This integration has five different dashboards available:
```

File: n2ws/README.md [hunk 8]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
...
+    'N2WSBackup&Recovery-Graphicalversion', 'N2WSBackup&Recovery-Graphicalversion-areas' and 'N2WSBackup&Recovery-EntitiesSpecificDashboard' for version 3.2.1
```
Comment (sarina-dd) [inline n2ws/README.md:43]:
```suggestion
    'N2WSBackup&Recovery-Graphicalversion', 'N2WSBackup&Recovery-Graphicalversion-areas' and 'N2WSBackup&Recovery-EntitiesSpecificDashboard' for N2WS Backup & Recovery v3.2.1. **Note**: These dashboards are only available for AWS users.
```

File: n2ws/README.md [hunk 9]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
...
+	'N2WSBackup&Recovery-EntitiesSpecificDashboardV4.1' and 'N2WSBackup&Recovery-GraphicalVersionV4.1' for version 4.1
```
Comment (sarina-dd) [inline n2ws/README.md:44]:
```suggestion
	'N2WSBackup&Recovery-EntitiesSpecificDashboardV4.1' and 'N2WSBackup&Recovery-GraphicalVersionV4.1' for N2WS Backup & Recovery v4.1. These dashboards are available for both AWS and Azure.
```

File: n2ws/README.md [hunk 10]
```diff
@@ -11,43 +12,49 @@ and more by Datadog monitoring service. This integration allows users to monitor
...
+ 
+    Alternatively users can [import JSON templates from N2WS][4].
```
Comment (sarina-dd) [inline n2ws/README.md:46]:
```suggestion
    Alternatively, you can [import JSON templates from N2WS][4] to create a dashboard.
```

**Review by sarina-dd — CHANGES_REQUESTED** [review]

File: n2ws/README.md (hunk 2 above)
Comment (eliadeini) [reply n2ws/README.md]: Hi, I've added a fix.

File: n2ws/README.md (hunk 3 above)
Comment (eliadeini) [reply n2ws/README.md]: Hi, I've added a fix.

File: n2ws/README.md (hunk 5 above)
Comment (eliadeini) [reply n2ws/README.md]: Hi, I've added a fix.

File: n2ws/README.md (hunk 7 above)
Comment (eliadeini) [reply n2ws/README.md]: Hi, I've added a fix.

File: n2ws/README.md (hunk 9 above)
Comment (eliadeini) [reply n2ws/README.md:46]: Hi, I've added a fix.

**Pull request closed without merging**